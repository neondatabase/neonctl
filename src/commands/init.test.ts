import { describe } from 'vitest';
import { test } from '../test_utils/fixtures';

describe('init', () => {
  test('init should run neon-init', async ({ testCliCommand }) => {
    await testCliCommand(['init']);
  });
});
