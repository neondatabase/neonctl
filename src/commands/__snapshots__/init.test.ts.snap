// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`init > init should run neon-init 1`] = `"{"npx-command":"neon-init","neon-init-args":[]}"`;

exports[`init > init with an argument 1`] = `"{"npx-command":"neon-init","neon-init-args":["--debug"]}"`;
